Index: kmess/CMakeLists.txt
===================================================================
--- kmess.orig/CMakeLists.txt	2009-02-22 17:29:15.000000000 +0200
+++ kmess/CMakeLists.txt	2009-02-22 17:44:00.000000000 +0200
@@ -42,7 +42,7 @@
 # so we need to ask QCA itself it it's got such a plugin. Pretty darned unportable stuff.
 IF( QCA2_FOUND )
   IF( UNIX )
-    EXECUTE_PROCESS( COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/cmake/check-qca2-ossl.sh OUTPUT_VARIABLE HAS_QCA2_OSSL_PLUGIN )
+    SET(HAS_QCA2_OSSL_PLUGIN 1)
     IF( HAS_QCA2_OSSL_PLUGIN EQUAL 1 )
     SET( QCA2_OSSL_PLUGIN_FOUND TRUE )
     ELSE( HAS_QCA2_OSSL_PLUGIN EQUAL 1 )
