Description: fix comments related to Debian package names
Author: Fathi Boudra <fabo@debian.org>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -25,8 +25,7 @@ IF( NOT KDE4_FOUND )
   "  \n"
   "  You will need this package and the matching development package.\n"
   "  Usually the package names for this library are:\n"
-  "    on Debian:   'kdebase-runtime' and 'kdelibs4-dev'\n"
-  "    on Ubuntu:   'kdebase-runtime' and 'kdelibs5-dev'\n"
+  "    on Debian and Ubuntu:   'kdebase-runtime' and 'kdelibs5-dev'\n"
   "    on openSUSE: 'kdebase4-runtime' and 'libkde4-devel'\n"
   "  \n"
   "  Please refer to the KMess board for more info,\n"
@@ -49,7 +48,7 @@ FIND_PACKAGE( LibXslt )
 MACRO_LOG_FEATURE( LIBXSLT_FOUND "LibXSLT" "A library to transform XML into other formats" "http://www.xmlsoft.org/XSLT" TRUE ""
     "Required for the chat styles system.
       * Usually the package names for this library are:
-        on Debian and Ubuntu: 'libxslt' and 'libxslt-dev'
+        on Debian and Ubuntu: 'libxslt' and 'libxslt1-dev'
         on openSUSE:         'libxslt' and 'libxslt-devel'
       * Please refer to the KMess board for more info,
         and specially to this guide: http://trac.kmess.org/wiki/Compiling%20KMess" )
@@ -63,7 +62,7 @@ IF( NOT GCRYPT_FOUND )
   MACRO_LOG_FEATURE( GCRYPT_FOUND "GCrypt" "A library to encrypt messages with various cyphers" "http://www.gnupg.org/" TRUE ""
     "It is required for the Live Messenger login process.
       * Usually the package names for this library are:
-        on Debian and Ubuntu: 'libgcrypt' and 'libgcrypt-dev'
+        on Debian and Ubuntu: 'libgcrypt' and 'libgcrypt11-dev'
         on openSUSE:          'libgcrypt' and 'libgcrypt-devel'
         on Fedora:            'gcrypt' and 'gcrypt-devel'
       * Please refer to the KMess board for more info,
